documentation/course/e2/README.md

```diff
@@ -1798,6 +1932,21 @@ TODO @tchojnacki: Dodać diagram pakietów, opis architektury.
 
 TODO @jakubzehner: Dodać diagram pakietów, opis architektury i endpointy.
 
+### API
+
+| **Rola**    | **Metoda** | **Endpoint**                   | **Wymagania** | **Opis**                                                                                                                |
+| ----------- | ---------- | ------------------------------ | ------------- | ----------------------------------------------------------------------------------------------------------------------- |
+| `guest`     | `GET`      | `/ext/v1/offers`               | TODO          | Pobranie listy dostępnych ofert biletowych.                                                                             |
+| `guest`     | `GET`      | `/ext/v1/offers/:id`           | TODO          | Pobranie informacji o ofercie biletowej.                                                                                |
```

Comment:
`guest`? We stated in E1 that we have to be logged in to see offers.

Reply:
True, only login and sign up are supported for guests.